src/vsr/replica.zig

```diff
@@ -2923,26 +3024,70 @@ pub fn ReplicaType(
         ///
         /// The caller owns the returned message, if any, which has exactly 1 reference.
         fn create_view_change_message(self: *Self, command: Command) *Message {
-            // We may send a start_view message in normal status to resolve a backup's view jump:
             assert(self.status == .normal or self.status == .view_change);
             assert((self.status == .normal) == (command == .start_view));
             assert((self.status == .view_change) == (command == .do_view_change));
-            assert(self.view >= self.log_view);
             assert(self.view >= self.view_durable());
             assert(self.log_view >= self.log_view_durable());
-
-            assert(command != .do_view_change or self.log_view < self.view);
-            assert(command != .start_view or self.log_view == self.view);
+            assert((self.log_view < self.view) == (command == .do_view_change));
+            assert((self.log_view == self.view) == (command == .start_view));
```

Comment:
Hopefully more readable. Maybe I should move these into the `switch (command)` but I kind of like having all of them together as preconditions.